--- PKGBUILD.orig	2016-10-01 12:31:52.315396761 +0200
+++ PKGBUILD	2016-10-01 17:14:34.794903727 +0200
@@ -3,13 +3,16 @@
 # Contributor: Bartłomiej Piotrowski <bpiotrowski@archlinux.org>
 # Contributor: Eivind Uggedal <eivind@uggedal.com>
 
+# ALARM: Kevin Mihelich <kevin@archlinuxarm.org>
+#  - armv7/aarch64 needs to be built with fPIC
+
 pkgname=mpv
 epoch=1
 pkgver=0.20.0
 pkgrel=1
 _waf_version=1.8.12
 pkgdesc='a free, open source, and cross-platform media player'
-arch=('i686' 'x86_64')
+arch=('i686' 'x86_64' 'armv6h' 'armv7h')
 license=('GPL')
 url='http://mpv.io'
 depends=(
@@ -35,6 +38,8 @@
 build() {
   cd ${pkgname}-${pkgver}
 
+  [[ $CARCH == "armv7h" || $CARCH == "aarch64" ]] && CFLAGS+=" -fPIC" && CXXFLAGS+=" -fPIC"
+
   ./waf configure --prefix=/usr \
     --confdir=/etc/mpv \
     --enable-cdda \
