--- PKGBUILD.orig	2017-11-26 16:03:15.964263522 +0100
+++ PKGBUILD	2017-11-26 16:03:55.380926156 +0100
@@ -3,13 +3,16 @@
 # Contributor: Bartłomiej Piotrowski <bpiotrowski@archlinux.org>
 # Contributor: Eivind Uggedal <eivind@uggedal.com>
 
+# ALARM: Kevin Mihelich <kevin@archlinuxarm.org>
+#  - armv7/aarch64 needs to be built with fPIC
+
 pkgname=mpv
 epoch=1
 pkgver=0.27.0
 pkgrel=4
 _waf_version=1.9.8
 pkgdesc='a free, open source, and cross-platform media player'
-arch=('x86_64')
+arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
 license=('GPL')
 url='https://mpv.io/'
 depends=(
@@ -50,6 +53,8 @@
 build() {
   cd ${pkgname}-${pkgver}
 
+  [[ $CARCH == "armv7h" || $CARCH == "aarch64" ]] && CFLAGS+=" -fPIC" && CXXFLAGS+=" -fPIC"
+
   ./waf configure --prefix=/usr \
     --confdir=/etc/mpv \
     --enable-cdda \
